src/Entity/IdeasWorkshop/ThreadComment.php

```diff
@@ -49,13 +49,13 @@
  *                 }
  *             }
  *         },
- *         "put_status_approve": {
+ *         "put_approval_toogle": {
  *             "method": "PUT",
- *             "path": "/thread_comments/{id}/approve",
+ *             "path": "/thread_comments/{id}/approval-toogle",
  *             "requirements": {"id": "%pattern_uuid%"},
  *             "denormalization_context": {"api_allow_update": false},
  *             "access_control": "object.getIdeaAuthor() == user",
- *             "controller": "AppBundle\Controller\Api\ThreadCommentController::approveAction",
+ *             "controller": "AppBundle\Controller\Api\ThreadCommentApprovalToogleController",
```

Comment:
No need of this controller. Just use a special `normalization_context` to change only `approved` property.